core/src/test/scala/org/apache/spark/sql/delta/DeltaCommitLockSuite.scala

```diff
@@ -38,6 +41,9 @@ class DeltaCommitLockSuite extends SparkFunSuite with LocalSparkSession with SQL
     spark = SparkSession.builder()
       .config("spark.delta.logStore.class", classOf[AzureLogStore].getName)
       .master("local[2]")
+      .config(StaticSQLConf.SPARK_SESSION_EXTENSIONS.key,
```

Comment:
Maybe create a until method in `DeltaTestUtils` to avoid repeating the same config code?